src/components/Confirm/index.js

```diff
@@ -57,7 +58,7 @@ const Previews = localised(({capture, retakeAction, confirmAction, error, method
     translate(`confirm.${documentType}.message`)
 
   return (
-    <div className={classNames(style.previewsContainer, {
```

Comment:
what was the bug on the confirmation screen?

Reply:
On iphone 5 you had to scroll down a bit to see the buttons. iPhone 5 screen is the smallest screen we support I believe so I've optimised in a way that the buttons will be fully visible on iPhone 5. On screens smaller than 320 x 568, the user might have to scroll down.

Before

![screenshot 2019-03-04 at 12 25 16](https://user-images.githubusercontent.com/7127427/53733921-584ec400-3e7a-11e9-9dfd-2b9ae5429b7e.png)

After

![screenshot 2019-03-04 at 12 31 13](https://user-images.githubusercontent.com/7127427/53733943-6270c280-3e7a-11e9-9ad5-01ecd2a61383.png)